src/www/ui/async/AjaxExplorer.php

```diff
@@ -402,6 +402,17 @@ private function createFileDataRow($child, $uploadId, $selectedAgentId, $pfileLi
     $isDecisionTBD = $this->clearingDao->isDecisionTBD($childUploadTreeId, $groupId);
     $img = $isDecisionTBD ? 'yellow' : $img;
 
+    // override green/red flag with greenRed flag in case of single file with decision type "Do Not Use"
+    $isDecisionDNU = $this->clearingDao->isDecisionDNU($childUploadTreeId, $groupId);
+    $img = $isDecisionDNU ? 'redGreen' : $img;
+
+    // override green/red flag with grey flag in case of no_license_found scanner finding
+    if (!empty($licenseList) && empty($editedLicenseList)) {
+      $noLicenseFound = LicenseDao::NO_LICENSE_FOUND;
+      $splitLicense = str_split($licenseList, strlen($noLicenseFound));
+      $img = (strcmp("No_license_found", $splitLicense[0]) <= 0) ? 'grey' : $img;
```

Comment:
In my opinion, will be better to use `!== 0` to get the exact match `No_license_found`.

Reply:
Instead of hard coding `"No_license_found"`, the variable `$noLicenseFound` can be reused.

Reply:
`strpos` will be better here instead of using `str_split` as you may get the required string split in between to two different elements.